requirements.txt

```diff
@@ -28,6 +28,7 @@ idna>=3.6
 # but latest 2.0 versions have an incompatibility with the requests library
 # See UN-1190.
 urllib3>=1.26.18,<2.0
+aiohttp>=3.10.5,<4.0
```

Comment:
Add aiohttp as dependency.
FWIW: Low version is what also happens to work with markdown-live-preview which we use for docs verification (among other things).